core/nuxeo-operation.js

```diff
@@ -199,15 +199,28 @@ import './nuxeo-connection.js';
     execute() {
       this._setActiveRequests(this.activeRequests + 1);
 
-      const params = !this.params || typeof this.params === 'object' ? this.params : JSON.parse(this.params);
+      let params = !this.params || typeof this.params === 'object' ? this.params : JSON.parse(this.params);
```

Comment:
Can `this.params` be `null`? If it can, `params` will be `null` too, which will be a problem when we try to `params.providerName` down in line 222.

Reply:
```
let params = !null || typeof null === 'object' ? null : JSON.parse(null); // undefined
```
This question still seems relevant.

Reply:
If the input is either a `page provider` or an element that extends the `page provider display behavior`, then params is not null, reason why we set the `params.providerName` for these cases.